### PR TITLE
[11.0][ADD] mrp_production_auto_post_inventory

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Add module mrp_production_auto_post_inventory to v11.0

This module allows the user to set to automatic the post inventory step
in a manufacturing order. Having this option activated, the inventory
will be automatically posted after the production has been done.

image

cc ~ @jbeficent

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
Looks good!

File: mrp_production_auto_post_inventory/models/company.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    mrp_production_auto_post_inventory = fields.Boolean(
+        string="Production Auto Post-Inventory",
+        help="Sets to automatic the post-inventory step in a manufacturing"
+             "order. The inventory will be posted right after the production"
+             "has been done")
```
Comment (LoisRForgeFlow) [inline mrp_production_auto_post_inventory/models/company.py]: The inventory is always posted when you mark as done the whole MO, so the description might be confusing. I would rather say "the inventory will be posted after some quantity is produced" or something similar.

I would correct this description in all the place it appears (readme, config, here...)

File: mrp_production_auto_post_inventory/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from . import company
+from . import res_config_settings
+# from . import mrp_production
```
Comment (HviorForgeFlow) [inline mrp_production_auto_post_inventory/models/__init__.py]: Remove unused imports

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1: minor comments

**Comment by JordiBForgeFlow** [comment]
@ageficent please squash the commits. This is ready to merge.

**Comment by AdriaGForgeFlow** [comment]
> @ageficent please squash the commits. This is ready to merge.

Done :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
:ok_hand:

**Comment by LoisRForgeFlow** [comment]
Travis error unrelated, merging.

**Pull request merged**